run_test.sh

```diff
@@ -154,6 +153,12 @@ function run_integration_test {
   export GOOGLE_CLOUD_PROJECT=datcom-website-dev
   export TEST_MODE=test
   python3 -m pytest -vv server/integration_tests/
+
```

Comment:
To confirm, will this exacerbate the `./run_test.sh -f` issue since that runs, and so, do we need to find a real fix for it?

Reply:
If so, might comment and make it manual until that's resolved.